bears/general/URLHeadBear.py

```diff
@@ -82,8 +83,9 @@ def check_prerequisites(cls):
     @staticmethod
     def get_head_response(url, timeout):
         try:
+            headers = {'User-Agent': UserAgent().random}
```

Comment:
And why are we using a random UserAgent? Using a specific browser header like firefox/chrome would be much better. 

On that note, why do we even need to generate a `user-agent`. Why not just hardcode the latest chrome header?